csharp/extractor/Semmle.Extraction.CSharp.DependencyFetching/IDotNetCliInvoker.cs

```diff
@@ -9,6 +10,18 @@ internal interface IDotNetCliInvoker
         /// </summary>
         string Exec { get; }
 
+        /// <summary>
+        /// A minimal environment for running the .NET CLI.
+        /// 
+        /// The .NET CLI language is set to English to avoid localized output.
+        /// </summary>
+        static ReadOnlyDictionary<string, string> MinimalEnvironment { get; } = new(new Dictionary<string, string>
+        {
+            {"DOTNET_CLI_UI_LANGUAGE", "en"},
+            {"MSBUILDDISABLENODEREUSE", "1"},
+            {"DOTNET_SKIP_FIRST_TIME_EXPERIENCE", "true"}
```

Comment:
Minor: The doc comment above explains `DOTNET_CLI_UI_LANGUAGE` but not the other two environment variables. Would it make sense to document why we are setting them as well? `DOTNET_SKIP_FIRST_TIME_EXPERIENCE` is kind of obvious, but `MSBUILDDISABLENODEREUSE` maybe less so.

Reply:
Will add some comments - then the tests will also be run a again 😄